tutorial/westeros/westeros_soft_constraints.ipynb

```diff
@@ -107,22 +106,25 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "df = pd.DataFrame({\n",
-    "    'node_loc': country,\n",
-    "    'technology': 'wind_ppl',\n",
-    "    'year_act': model_horizon,\n",
-    "    'time': 'year',\n",
-    "    'value': [.00, .0, .0, .01],\n",
-    "    'unit': '-'})\n",
-    "scen.add_par('soft_activity_up', df)"
+    "df = pd.DataFrame(\n",
```

Comment:
In some cases like here, `black` was applied but the underlying code idiom was left as-is. The code works and it's fine to merge, but I think a later PR could improve these further to suggest a common style, e.g.:
```py
name = "soft_activity_up"
scen.add_par(
    name,
    make_df(
        name,
        node_loc=country,
        technology="wind_ppl",
        year_act=model_horizon,
        time="year",
        value=[0, 0, 0, 0.01],
        unit="-",
    )
)
```
or
```py
name = "soft_activity_up"
data = make_df(
    name,
    node_loc=country,
    technology="wind_ppl",
    year_act=model_horizon,
    time="year",
    value=[0, 0, 0, 0.01],
    unit="-",
)
scen.add_par(name, data)
```